### PR TITLE
Create l-switchbasemap and l.multiControl plugins

### PR DESCRIPTION
**Pull request by clavijojuan**

### CODE REVIEW

File: docs/_plugins/layer-switching-controls/l-switchasemap.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: L.switchBasemap
+category: layer-switching-controls
+repo: https://github.com/clavijojuan/L.switchBasemap
+author: clavijojuan
+author-url: https://github.com/clavijojuan
+demo: https://elegant-meninsky-515912.netlify.app/
+compatible-v0:
```
Comment (Falke-Design) [inline docs/_plugins/layer-switching-controls/l-switchasemap.md:8]: I looks like you are not compatible with v0 from Leaflet. Please add `false`
```suggestion
compatible-v0: false
```

File: docs/_plugins/layer-switching-controls/l-switchasemap.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: L.switchBasemap
+category: layer-switching-controls
+repo: https://github.com/clavijojuan/L.switchBasemap
+author: clavijojuan
+author-url: https://github.com/clavijojuan
+demo: https://elegant-meninsky-515912.netlify.app/
```
Comment (Falke-Design) [inline docs/_plugins/layer-switching-controls/l-switchasemap.md:7]: I'm against merging this PR until attribution is displayed in the demo. Tile hosters want to have a copyright notice while usage: https://www.openstreetmap.org/copyright

The attribution should also changed while switching the base layer.

grafik

**Review by Falke-Design — CHANGES_REQUESTED** [review]

File: docs/_plugins/layer-switching-controls/l-switchasemap.md (hunk 2 above)
Comment (clavijojuan) [reply docs/_plugins/layer-switching-controls/l-switchasemap.md:7]: Hi, fixed. Sorry for that.

**Review by Falke-Design — APPROVED** [review]

**Comment by clavijojuan** [comment]
Hi, somebody can tell me why dont merge? its wrong or has fixes to do? 😢 

Thanks

**Review by Malvoz — APPROVED** [review]

**Review by Falke-Design — APPROVED** [review]

**Pull request merged**